tests/fs.md

```diff
@@ -495,3 +499,25 @@ Unconfined:
 +rename <fs:../foo> to <fs:foo> -> ok
 - : unit = ()
 ```
+
+# Stat
```

Comment:
With `make test_luv`, I get:
```
+|Exception: Eio.Fs.Not_found ("stat_link", _)
```

Reply:
oopsy, will fix it.

Reply:
heh, `File.realpath` resolves the symlink and returns ENOENT, fix is not trivial, but since we don't do it in u-ring and therefore behaves differently, this is a bug. We're not using realpath(3) in Uring.

Reply:
I guess we should just do the `realpath` on the parent, then? It's fine to lstat a symlink even if we don't have access to its target.

Reply:
Oh that's smart, I hadn't considered it.

Reply:
I won't address the `realpath` on the parent here, it's outside of the scope imho.

Reply:
Yes, now `Path.stat` is gone - we can worry about that later :-) PR title needs updating too...